### PR TITLE
cockroach refactor

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

cockroaches are now a hostile mob and glockroaches are a child
also glockroaches dont 1 shot tables and stuff anymore

## Why It's Good For The Game

copypaste BAD??

## Changelog
:cl:
refactor: glockroach
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
wtf i hate deltastation2.dmm now?

File: code/modules/mob/living/simple_animal/hostile/cockroach.dm [hunk 1]
```diff
@@ -28,25 +19,49 @@
...
+	obj_damage = 0
 	ventcrawler = VENTCRAWLER_ALWAYS
-	gold_core_spawnable = HOSTILE_SPAWN
+	gold_core_spawnable = FRIENDLY_SPAWN
 	verb_say = "chitters"
 	verb_ask = "chitters inquisitively"
 	verb_exclaim = "chitters loudly"
 	verb_yell = "chitters loudly"
+	var/squish_chance = 50
+	del_on_death = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/cockroach.dm]:
```suggestion
	del_on_death = TRUE
```

File: code/modules/mob/living/simple_animal/hostile/cockroach.dm [hunk 2]
```diff
@@ -28,25 +19,49 @@
...
+	melee_damage_upper = 0
+	obj_damage = 0
 	ventcrawler = VENTCRAWLER_ALWAYS
-	gold_core_spawnable = HOSTILE_SPAWN
+	gold_core_spawnable = FRIENDLY_SPAWN
 	verb_say = "chitters"
 	verb_ask = "chitters inquisitively"
 	verb_exclaim = "chitters loudly"
 	verb_yell = "chitters loudly"
+	var/squish_chance = 50
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/cockroach.dm]: Move this to the bottom

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Fikou** [comment]
FUCK

**Pull request merged**